### PR TITLE
dev: clean commands

### PR DESCRIPTION
**Pull request by scop**
For commands consistency.

EDIT(ldez): I rewrote some commands to be more idiomatic and consistent.

### CODE REVIEW

File: pkg/commands/version.go [hunk 1]
```diff
@@ -33,7 +33,10 @@ func (e *Executor) initVersion() {
 	versionCmd := &cobra.Command{
 		Use:   "version",
 		Short: "Version",
-		RunE: func(cmd *cobra.Command, _ []string) error {
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) != 0 {
+				e.log.Fatalf("Usage: golangci-lint version")
```
Comment (kamilsk) [inline pkg/commands/version.go:38]: strange pattern https://github.com/golangci/golangci-lint/search?q=len%28args%29+%21%3D+0. why not `Args: cobra.NoArgs,` ?

**Review by kamilsk — APPROVED** [review]

File: pkg/commands/version.go (hunk 1 above)
Comment (ldez) [reply pkg/commands/version.go]: You are right, I rewrote all the commands.

**Review by ldez — APPROVED** [review]
LGTM.
I took the opportunity to rewrite a bit the commands

File: pkg/commands/version.go (hunk 1 above)
Comment (kamilsk) [reply pkg/commands/version.go]: awesome!!!

**Pull request merged**